app/Models/City.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class City extends Model
+{
+    protected $fillable = [
+        'name'
+    ];
```

Comment:
instead of adding the fillables, sometimes it is better to add just the guarded instead.. otherwise each time you add a new field you will have to edit the model.

```suggestion
    protected $guarded = [
        'id'
    ];
```